walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,62 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    //some basic if statements
+    if(request.getAttendees().isEmpty()){
+        //when there are no attendees
+        return Arrays.asList(TimeRange.WHOLE_DAY);
+    } else if(request.getDuration() > TimeRange.END_OF_DAY) {
+        //when the meeting duration is longer than a day
+        return Arrays.asList();
+    }
```

Comment:
you can just write a single comment at the top of the if block. something like 
"handle edge cases: if there are no attendees, then the time range is the whole day and if the request duration is longer than a day, then there's no valid time range"

Reply:
Just fixed, thanks!